### DISCUSSION TITLE
Ring Doorbell 2. Gen (optional Battery and wired power) - manual snapshot blocked by recording

### DISCUSSION THREAD

**Discussion by jellyneck84** [tsightler/ring-mqtt#769] [Support Q&A]
Hello @tsightler ,
I'm using the doorbell model which can be used with battery or wired. It's connected to my wiring (doorbell transformator) and therefore always 100% loaded.

I have the problem, that without a subscription, there is no image URL in the motion detection which can be used for taking a snapshot. 
If I'm trying to snapshot with an automation on ding (ringing), it's not working because snapshot is blocked by recording:
Received motion push notification, expires in 10 seconds
2023-12-27T19:31:21.806Z ring-mqtt [Haustür] The motion notification did not contain image UUID and battery cameras are unable to snapshot while recording

If i am right, i can't disable motion detection, as i am not able to do a manual snapshot then.

Is there any way to disable the "recording", i don't need that, i just want to trigger a snapshot when there is a ding event. 
Additional, i don't understand the "event select" dropdown on the main page of the integration, what exactly is it doing? Did not understand the documentation for that.

**Comment by tsightler** [comment]
Without a subscription there's basically no practical method to do what you want.  I feel these limitations are already well described in the documentation, which you effectively communicated in your post, so I'm not sure what more there is to add.  A low-power camera can't take snapshots while recording and you are right, of course you can disable motion detection, but then I'm not sure how you would know when a snapshot needs to be taken.  There is no practical way to get a notification snapshot for these cameras without a subscription, period.

Some users have posted automation scripts which start a live stream and extract an image from that, this is of course possible, but the image will typically be delayed by 5-10 seconds since it takes time to start the stream and extract an image, but it is the only other options other than simply paying the subscription.

Even select doesn't do anything for you since you don't have a subscription, it selects the event recording from the event history that would be played back by the event stream camera, but, without a subscription, there are no recorded event history, so it doesn't nothing.  The documentation mentions that this feature requires a subscription.

**Reply by jellyneck84** [reply]
Hallo @tsightler ,

Its easy to know when to take a snapshot, because there is the Ding Event. So i would not need the motion Event. 

Thanks for clarifying, lets See if i need to switch to another Model.

**Reply by jellyneck84** [reply]
Just a quick solution for others maybe..... it's not possible to take a snapshot at this moment, but it's fine to capture a video . So i decided to capture a 3 second video on the ding event and send it to telegram.

alias: Send video to telegram on ding
description: ""
trigger:
  - platform: state
    entity_id:
      - camera.haustur_snapshot
      - binary_sensor.haustur_ding
    to: "on"
condition: []
action:
  - service: camera.record
    data:
      filename: /config/www/kamerafoto/haustuer_video.mp4
      duration: 3
    target:
      entity_id: camera.homeassistant
  - service: telegram_bot.send_video
    data:
      authentication: digest
      target: 6xxxxxx57xx (your telegram ID)
      file: /config/www/kamerafoto/haustuer_video.mp4
      parse_mode: markdownv2
      caption: Jemand ist an der Haustür
mode: single
